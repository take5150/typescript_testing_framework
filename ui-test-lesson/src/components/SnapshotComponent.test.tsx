import { render } from "@testing-library/react";
import SnapshotComponent from "./SnapshotComponent";

describe("SnapshotComponent", () => {
  it("should render text", () => {
    // ? containerである必要は？
    const { container } = render(<SnapshotComponent text={"test"} />);
    expect(container).toMatchSnapshot();
  });
});
