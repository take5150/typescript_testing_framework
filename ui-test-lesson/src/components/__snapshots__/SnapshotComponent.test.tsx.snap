// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotComponent should render text 1`] = `
<div>
  <div>
    <p>
      test
    </p>
  </div>
</div>
`;
